rpcs3/Emu/System.cpp

```diff
@@ -935,7 +936,7 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		sys_log.notice("Title: %s", GetTitle());
 		sys_log.notice("Serial: %s", GetTitleID());
 		sys_log.notice("Category: %s", GetCat());
-		sys_log.notice("Version: %s / %s", GetAppVersion(), version_disc);
+		sys_log.notice("Version: APP_VER=%s VERSION=%s", version_app, version_disc);
```

Comment:
I think this will impact on the discord bot parsing
https://github.com/RPCS3/discord-bot/blob/363d93545605723a0f8964100820ec5bf2957af1/CompatBot/EventHandlers/LogParsing/LogParser.LogSections.cs#L45